tasks/install_shell.sh

```diff
@@ -525,6 +525,10 @@ do_download "$download_url" "$download_filename"
 
 install_file $filetype "$download_filename"
 
+if [[ $PT_stop_service = true ]] && [ $platform != "mac_os_x" ]; then
+  /opt/puppetlabs/bin/puppet resource service puppet-agent ensure=stopped enable=false
```

Comment:
I think it is OK to unconditionally stop the service. My comment above was more about updating the spec test to allow for `ensure => absent` in the service command result for osx (and window maybe I cant remember if it is started there).